manager/controllers/utils/utils.go

```diff
@@ -115,21 +115,6 @@ func SupportsInterface(array []*app.InterfaceDetails, element *app.InterfaceDeta
 	return false
 }
 
-// GetModuleCapabilities checks if the requested capability is supported by the module.  If so it returns
-// the ModuleCapability structure.  There could be more than one, since multiple structures could exist with
-// the same CapabilityType but different protocols, dataformats and/or actions.
-func GetModuleCapabilities(module *app.FybrikModule, requestedCapability string) (bool, []app.ModuleCapability) {
```

Comment:
sidenote: SupportsInterface is also unused.

Reply:
ok